src/transformState.js

```diff
@@ -1,11 +1,28 @@
+/* eslint-disable no-unused-vars */
 'use strict';
 
 /**
  * @param {Object} state
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Maybe here is better to give a conscious name to variable.